ibm/resource_ibm_satellite_cluster_worker_pool.go

```diff
@@ -229,15 +229,11 @@ func resourceIBMSatelliteClusterWorkerPoolRead(d *schema.ResourceData, meta inte
 	d.Set("worker_pool_labels", IgnoreSystemLabels(workerPool.Labels))
 	d.Set("host_labels", flattenWorkerPoolHostLabels(workerPool.HostLabels))
 
-	var zones = make([]map[string]interface{}, 0)
-	for _, zone := range workerPool.Zones {
-		zoneInfo := map[string]interface{}{
-			"id": *zone.ID,
-		}
-		zones = append(zones, zoneInfo)
+	if v, ok := d.GetOk("zones"); ok {
```

Comment:
same as above

Reply:
Removed GetOk method and set the zones parameter in the imported section.